corgi/core/migrations/0041_auto_20221024_1844.py

```diff
@@ -43,4 +43,25 @@ class Migration(migrations.Migration):
             name="type",
             field=models.CharField(choices=[("BREW", "Brew"), ("KOJI", "Koji")], max_length=20),
         ),
+        migrations.RunSQL("DROP INDEX IF EXISTS core_compon_latest_name_type_idx"),
+        migrations.RunSQL("DROP INDEX IF EXISTS core_compon_latest_type_name_idx"),
+        migrations.RunSQL("DROP INDEX IF EXISTS core_compon_latest_idx"),
+        migrations.RunSQL(
+            "CREATE INDEX core_compon_latest_name_type_idx ON public.core_component \
+             USING btree (name, type) WHERE (((type)::text = 'SRPM'::text) OR \
```

Comment:
```suggestion
             USING btree (name, type) WHERE ((((type)::text = 'RPM'::text) AND ((arch)::text = 'src')) OR \
```

Reply:
Jason and I discussed this over chat, he pushed a new commit that turns these into model indexes. See latest commit.